detectable/src/test/resources/detectables/functional/cargo/dependencyHierarchy.lock

```diff
@@ -0,0 +1,18 @@
+[[package]]
```

Comment:
If you are able to have the CargoLockParser take in a `String` instead of an `InputStream` then these resource files should removed in favor of a `String` within the unit test.